deployments/roman/environments/roman-cal/roman-phase1.pip

```diff
@@ -20,4 +19,5 @@ bokeh>=3.4.1
 #astroquery
 pandas>=2.2.2
 photutils>=1.12.0
-s3fs>=2024.3.1
\ No newline at end of file
+s3fs>=2024.3.1
+xarray
```

Comment:
I think xarray and s3fs need to be installed in the kernel where jdaviz is installed, not in romancal